src/Type/Object/ThemeMods.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace WPGraphQL\Type;
+
+use WPGraphQL\Data\DataSource;
+
+register_graphql_object_type( 'ThemeMods', [
+	'description' => __( 'All of registered theme modifications' ),
+	'interfaces'  => [ WPObjectType::node_interface() ],
+	'fields'  => [
+		'background' => [ 
+			'type' 			=> 'MediaItem',
+			'description'	=> __( 'custom background' ),
+			'resolve'		=> function( $root, $args, $context, $info ) {	
+				if( ! empty( $root['background'] ) ) { 
+					return ( ! empty( $root['background']['id'] ) ) ?
+					DataSource::resolve_post_object( absint( $root['background']['id'] ), 'attachment' ) :
+					null;
+				}
+
+				return null;
+			}
+		],
+		'backgroundColor' => [ 
+			'type' 			=> 'String',
+			'description'	=> __( 'background color' ),
+			'resolve'		=> function( $root, $args, $context, $info ) {
+				return ( ! empty( $root['background_color'] ) ) ? $root['background_color'] : null;
+			}
+		],
+		'customCssPostId' => [ 
+			'type' 			=> 'Int',
+			'description'	=> __( 'custom theme logo' ),
+			'resolve'		=> function( $root, $args, $context, $info ) {
+				return ( ! empty( $root['custom_css_post_id'] ) ) ? $root['custom_css_post_id'] : null;
+			}
+		],
+		'customLogo' => [ 
+			'type' 			=> 'Int',
+			'description'	=> __( 'WP ID of customLogo mediaItem' ),
+			'resolve'		=> function( $root, $args, $context, $info ){
+				return ( ! empty( $root['custom_logo'] ) ) ? absint( $root['custom_logo'] ) : null;
+			}
+		],
+		'headerImage' => [ 
+			'type' 			=> 'MediaItem',
+			'description'	=> __( 'custom header image' ),
+			'resolve'		=> function( $root, $args, $context, $info ){
+				if( ! empty ( $root['header_image'] ) ) {
+					return ( ! empty( $root['header_image']['id'] ) ) ?
+					DataSource::resolve_post_object( absint( $root['header_image']['id'] ), 'attachment' ) :
+					null;
+				}
+				
+				return null;
+			},
+		],
+		'navMenu' => [
```

Comment:
Edge data in `header_image` and `background_image` mods stored and can be updated in mutations but can not be queried at the moment.